roles/nvidia-gpu-tests/tasks/main.yml

```diff
@@ -0,0 +1,86 @@
+---
+- name: Get a count of GPUs
+  shell: "lspci -d 10de: | grep -c VGA | wc -l"
+  register: gpu_count
+
+# This bit seems a bit buggy depending on the OS 
+- name: Install cuda toolkit
+  become: yes
+  package:
+    name: nvidia-cuda-toolkit
+    state: present
+  when: gpu_test_install_toolkit
+
+- name: download cuda samples
+  git:
+    repo: https://github.com/NVIDIA/cuda-samples.git
+    clone: yes
+    dest: "{{ gpu_test_cuda_dir }}"
+  when: gpu_test_build
+  ignore_errors: yes
+
+- name: make p2p
+  shell: "cd {{ gpu_test_samples_dir }}/p2pBandwidthLatencyTest && make"
```

Comment:
This doesn't run for me. Looks like an issue with the cuda-samples?

```
TASK [nvidia-gpu-tests : make p2p] **********************************************************************************************************************************
fatal: [deepops-dgx1q-04]: FAILED! => changed=true
  cmd: cd /tmp/cuda-samples/Samples/p2pBandwidthLatencyTest && make
  delta: '0:00:00.257728'
  end: '2020-01-28 03:17:56.087630'
  msg: non-zero return code
  rc: 2
  start: '2020-01-28 03:17:55.829902'
  stderr: |-
    make: /usr/local/cuda/bin/nvcc: Command not found
    make: *** [p2pBandwidthLatencyTest.o] Error 127
  stderr_lines: <omitted>
  stdout: |-
    /usr/local/cuda/bin/nvcc -ccbin g++ -I../../Common  -m64    -gencode arch=compute_30,code=sm_30 -gencode arch=compute_35,code=sm_35 -gencode arch=compute_37,code=sm_37 -gencode arch=compute_50,code=sm_50 -gencode arch=compute_52,code=sm_52 -gencode arch=compute_60,code=sm_60 -gencode arch=compute_61,code=sm_61 -gencode arch=compute_70,code=sm_70 -gencode arch=compute_75,code=sm_75 -gencode arch=compute_75,code=compute_75 -o p2pBandwidthLatencyTest.o -c p2pBandwidthLatencyTest.cu
    Makefile:290: recipe for target 'p2pBandwidthLatencyTest.o' failed
  stdout_lines: <omitted>
...ignoring
```

Reply:
Yeah, there are actually some issues with this building by default on a DGX, because of how nvcc is installed.